src/main/java/com/redhat/lightblue/healthcheck/HealthCheckResource.java

```diff
@@ -44,17 +49,77 @@
 
     private static final String ENTITY = "test";
 
-    private final LightblueClient client;
+    private static final String CLIENT_ERROR_DELIMITER = " , ";
```

Comment:
What about just letting the comma be the delimiter and calling the trim method of the file paths?